docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -1369,24 +1369,6 @@ As mentioned, use `AccessTokenRequestFilter` if you work with Keycloak or an Ope
 You can generate the tokens as described in xref:security-oidc-bearer-token-authentication.adoc#bearer-token-integration-testing[OpenID Connect Bearer Token Integration testing] section.
 Prepare the REST test endpoints. You can have the test front-end endpoint, which uses the injected MP REST client with a registered token propagation filter, call the downstream endpoint. For example, see the `integration-tests/resteasy-client-oidc-token-propagation` in the `main` Quarkus repository.
```

Comment:
Note for this PR but please let's not point to random content in our repo without even giving a link.

Please include some code here as an example.

Reply:
Hi @gsmet Sure, I must've typed it with the very initial version of this doc, agree it is a bad idea.
How about linking to the corresponding quickstart section, https://quarkus.io/guides/security-openid-connect-client#testing-the-application ? There is a lot of code involved, and the quickstart covers it all for OIDC client and propagation.
If you agree, I can push a commit to your PR adding it.

Reply:
So we replace `integration-tests/resteasy-client-oidc-token-propagation` with a link. Let me know what you prefer

Reply:
Or I can follow up a bit later with  dedicated PR, whatever you prefer works for me

Reply:
Yes, please do a follow-up!